overlord/snapstate/snapstate_update_test.go

```diff
@@ -14608,10 +14608,23 @@ func (s *snapmgrTestSuite) testRevertWithComponents(c *C, undo bool) {
 		c.Assert(chg.Err(), IsNil, Commentf("change tasks:\n%s", printTasks(chg.Tasks())))
 	}
 
+	var expected fakeOps
+	for _, cs := range seq.ComponentsForRevision(prevSnapRev) {
```

Comment:
@pedronis these changes here to `testRevertWithComponents` check that we are running `validate-component` with `SkipAssertionsDownload == true`.